src/clj/commiteth/config.clj

```diff
@@ -1,10 +1,16 @@
 (ns commiteth.config
   (:require [cprop.core :refer [load-config]]
             [cprop.source :as source]
-            [mount.core :refer [args defstate]]))
+            [mount.core :refer [args defstate] :as mount]))
 
 (defstate env :start (load-config
                        :merge
                        [(args)
                         (source/from-system-props)
                         (source/from-env)]))
+
```

Comment:
Here and below, it would be great to allow a developer run exactly those subsystems that he or she really need when working on a task. While working on that task, I didn't manage to run some of that due to configuration issues. Instead, I just launched config, db and http-server which were enough.

Reply:
How about describing the different combinations/groups that could be useful instead of adding a fn to start/stop states independently? In the end starting/stopping states independently can be done easily in the REPL. For example we could have a group to start just the web app without any connection to Ethereum (and without things that depend on that).